src/actions/AuthActions.js

```diff
@@ -15,23 +18,82 @@ import {tokens, navKeys} from '../constants';
 
 import {persistor} from '../store';
 
-const userLoginSuccess = (dispatch, email, navigate) => {
+const userLoginSuccess = ({dispatch, user, navigate}) => {
     dispatch({
         type: FACEBOOK_LOGIN_SUCCESS,
-        payload: {token: email, appReady: false}
+        payload: {token: user.email, currentUser: user, appReady: false}
```

Comment:
We now pass the concept of `currentUser` all over the app. We can probably deprecate `token` soon.